### PR TITLE
Grammar and suggestions for reference-article.txt

### PR DESCRIPTION
**Pull request by luxuryluke**
Various changes to make the Files page more understandable to me. But some of this may not match your site style guides.

### CODE REVIEW

File: content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt [hunk 1]
```diff
@@ -81,48 +81,48 @@ gallery:
   query: page.images
 ```
 
-You can use all file types available in Kirby here (`images`, `documents`, `videos`, `audio` and `code`).
+You can use all file types available in Kirby here (`images`, `documents`, `videos`, `audio`, and `code`).
 
 ### Querying files from other pages
 
-You can get as complex as you like, e.g. get all images of all children of the `photography` page that use the `cover` template:
+You can get as complex as you like, e.g. 'get all images from all children of the `photography` page that use the `cover` template:
```
Comment (texnixe) [inline content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt]: What's this opening quote doing here?

**Comment by distantnative** [comment]
This is hard to merge, cause it adds a lot of things that are very out of style for our docs (wrapping terms with `'` that are very standard, adding a lot of `the` in places where it's not needed), while other things are surely improving the text.

File: content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@ Title: Files
 
 Intro:
 
-A files select field that allows to select one or multiple related files
+A 'files' 'select' field that allows selecting one or multiple related files
```
Comment (texnixe) [inline content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt]: These two quoted words seem weird. And we are not using single quotes anywhere else for input fields, so I wouldn't start this here, either.

File: content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt [hunk 3]
```diff
@@ -14,7 +14,7 @@ Guide: content/files
 
 Text:
 
-The files field allows you to select one or more files.
+The 'files' field allows you to select one or more files.
```
Comment (texnixe) [inline content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt]: Same here.

**Comment by luxuryluke** [comment]
Okay, I'll strip out the single quotes and 'the' instances.

File: content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt [hunk 4]
```diff
@@ -134,19 +134,19 @@ image:
 
 ### No image
 
-Set the image options to false if you don't want to show an image but an icon instead:
+Don't want to see an image preview? Set the image options to 'false' if you don't want to show an image, which will show an icon instead:
```
Comment (texnixe) [inline content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt]: This should be `false`, not quoted.

File: content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt [hunk 5]
```diff
@@ -134,19 +134,19 @@ image:
...
 
 ```yml
 image: false
 ```
 
 ## Upload options
 
-By default, the files field allows you to upload files. You can define the `Add` button behavior with the `uploads` option: When clicking on the `Add` button, you can either select a file and/or upload a file (which is then automatically selected).
+By default, the files field allows you to upload files. You can define the `Add` button behavior with the `uploads` option: When clicking on the `Add` button, you can either 'select a file' and/or 'upload a file' (which is then automatically selected).
```
Comment (texnixe) [inline content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt]: Same here with the quotes.

**Comment by texnixe** [comment]
Thanks for your suggestions, apart from my comments, this is great. Could you please remove all the quotes again, please?

**Comment by distantnative** [comment]
@texnixe I think you forgot to submit the comments.

**Comment by texnixe** [comment]
> @texnixe I think you forgot to submit the comments.

Looks like it ;). Anyway, you were faster with your comment while I was adding my single comments...

File: content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt (hunk 1 above)
Comment (luxuryluke) [reply content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt]: Unfinished idea. Delete it.

File: content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt (hunk 3 above)
Comment (luxuryluke) [reply content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt]: Remove 'em!

**Comment by luxuryluke** [comment]
You're both so fast. Thanks for the help!

**Comment by distantnative** [comment]
Sorry, shouldn't have commented on them, should've known that you're on it anyways since they're docs-related.

File: content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt (hunk 5 above)
Comment (luxuryluke) [reply content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt]: Let's strip these out, then. My brain was speaking with air quotes. 👍

**Comment by luxuryluke** [comment]
Okay, as I went in to delete some of these changes in my patch they were already made? I'll wait a bit and see if you two are already fixing this or not. Editing code on github.dev is not how I like to code, lol.
I'll revisit this in a bit and see if it needs fixing. Cheers!

**Comment by texnixe** [comment]
> if you two are already fixing this or not

I have done nothing 😇

**Comment by distantnative** [comment]
Me neither 🤷‍♂️

**Review by luxuryluke — COMMENTED** [review]
Changes made.

**Comment by texnixe** [comment]
Great, thanks @luxuryluke 💛

**Pull request merged**